vyper/parser/stmt.py

```diff
@@ -108,12 +108,18 @@ def _check_valid_assign(self, sub):
                 return
             else:
                 raise TypeMismatchException('Invalid type, expected: bytes32', self.stmt)
+        # This is dead code once anonymous structs are removed.
```

Comment:
This code can be removed then, right? :hocho:

Reply:
Fixed in 9e7bdac9fef9c737b028b223fcdf0d7f715ede82